public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -108,14 +108,13 @@ function ModalMissionComplete (svl, missionContainer, missionModel, taskContaine
         if (svl.missionsCompleted === 2) {
             // Load the validation page since they've done 2 missions.
             window.location.replace('/validate');
-        }
-        else if (svl.neighborhoodModel.isNeighborhoodCompleted) {
+        } else if (svl.neighborhoodModel.isNeighborhoodCompleted) {
             // Reload the page to load another neighborhood.
             window.location.replace('/audit');
         } else {
             // TODO can we require that we have a new mission before doing this?
             var nextMission = missionContainer.getCurrentMission();
-            _modalModel.triggerMissionCompleteClosed( { nextMission: nextMission } );
```

Comment:
It looks like maybe you have some setting turned on in your editor that automatically updates the code style? Could you turn that off? Although I agree with most of the style changes, it distracts from the code you are actually trying to change.